src/resources/views/crud/fields/checklist.blade.php

```diff
@@ -25,11 +25,19 @@
 
 @include('crud::fields.inc.wrapper_start')
     <label>{!! $field['label'] !!}</label>
+    <div class="row">
```

Comment:
We should probably hide this "Select all" option by default, and only show it if the developer has defined `'show_select_all' => true`. That way, it's a non-breaking change, and devs with very few entries in the checklist are not affected.